### DISCUSSION TITLE
Wandthermostat-mit-Luftfeuchtigkeitssensor als separater Feuchtigkeit Entität

### DISCUSSION THREAD

**Discussion by arne182** [SukramJ/aiohomematic#1150] [Ideas]
Ich habe gemerkt ich habe keine zugriff auf den verlauf von den Feuchtigkeitssensor von meine Wandthermostat. 
Wäre es möglich den als sensor einzurichten? Aktuell sehe ich nur Diese Entität ist nicht verfügbar wenn ich in meine Wandthermostat mit Luftfeuchtigkeitssensor auf den Sensoren rein klick.
Für meinen temperatur-und-luftfeuchtigkeitssensor-aussen geht es über den Sensoren in HA Einwand frei.

**Comment by SukramJ** [comment]
Solche Fragen lassen sich leicht über die [Doku](https://github.com/danielperna84/custom_homematic#deactivated-entities) klären.

**Reply by arne182** [reply]
Geht es schon? kannst du mir zeigen wo? Ich sehe es nicht in die Doku

**Reply by SukramJ** [reply]
Wenn du auf den Link klickst wird die richtige Stelle (deactivated entities) geöffnet.

**Reply by arne182** [reply]
Ich habe es gefunden. in die Doku stand etwas mit Levels und unignore in eine Datei ändern.
Es war aber so leicht wie versteckte Sensoren einfach zu Aktivieren. Wäre schön eine Leichten Hinweis in die Doku zu haben aber der Noob muss erstmal lernen das man Sensoren deaktivieren und aktivieren kann

**Reply by SukramJ** [reply]
Das sind Home Assistant Basics. 
Wir haben den Punkt zu den deaktivierten Entitäten hier nur zusätzlich aufgenommen, weil einige Benutzer nicht experimentierfreudig sind.

Wenn Du mit sinnvollen Verbesserungen beitragen möchtest, dann kannst Du gerne einen Pull Request für die [README.md](https://github.com/danielperna84/custom_homematic/blob/devel/README.md) mit deinen Vorschlägen erstellen.
